torchci/clickhouse_queries/oss_ci_benchmark_v3/query.sql

```diff
@@ -0,0 +1,104 @@
+-- This query creates the oss_ci_benchmark_v3 table on ClickHouse
+CREATE TABLE benchmark.oss_ci_benchmark_v3 (
+    -- Metadata
+    `timestamp` UInt64,
+    `schema_version` String DEFAULT 'v3',
+    `name` String,
+    -- About the change
+    `repo` String DEFAULT 'pytorch/pytorch',
+    `head_branch` String,
+    `head_sha` String,
+    `workflow_id` UInt64,
+    `run_attempt` UInt32,
+    `job_id` UInt64,
+    -- (optional) Service Lab id if the result comes from there
+    `servicelab_experiment_id` UInt64 DEFAULT '0',
+    `servicelab_trial_id` UInt64 DEFAULT '0',
+    -- The raw records on S3
+    `s3_path` String,
+    -- About the device where the benchmark runs:
+    --  name, the optional name of the runner
+    --  type, the runner type or label, i.e. linux.aws.a100, or android S22
+    --  cpu_info, i.e. x86_64 or arm
+    --  cpu_count, the number of CPU cores
+    --  mem_info, TBD
+    --  avail_mem_in_gb, the amount of available memory (GB)
+    --  gpu_info, i.e. NVIDIA A100 or mps
+    --  gpu_count, the number of GPU devices
+    --  gpu_mem_info, TBD
+    --  avail_gpu_mem_in_gb, the total amount of available GPU memory (GB)
+    --  extra_info, any extra piece of information in key/value format
+    `runner` Tuple(
+        name String,
+        type String,
+        cpu_info String,
+        cpu_count UInt32,
+        mem_info String,
+        avail_mem_in_gb UInt32,
+        gpu_info String,
+        gpu_count UInt32,
```

Comment:
@Jack-Khuu Just FYI, I think that keeping the gpu_info and gpu_count here would capture the case of having more than one GPUs in one runner.  My assumption is that all GPUs on the same runner will be the same.  For distributed, there might be more than one runners I guess, so let me make this a list of runners instead to capture that